### PR TITLE
EN6-53 Bug Corrections

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/contentsettings/ContentSettingsService.java [hunk 1]
```diff
@@ -203,6 +207,54 @@ private void saveSystemParams(Map<String,String> systemParams) {
         return getEditor();
     }
 
+    private void validateMetadata(Map<String,List<String>> metadata, String key, String mapping) {
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/contentsettings/ContentSettingsService.java]: Does this have unit tests?

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/contentsettings/ContentSettingsService.java [hunk 2]
```diff
@@ -203,6 +207,54 @@ private void saveSystemParams(Map<String,String> systemParams) {
...
+            }
+        }
+
+        if (metadata.containsKey(key)) {
+            errors.reject("4", null, "contentsettings.metadata.duplicate");
+            throw new ValidationConflictException(errors);
+        }
+    }
+
+    private void validateCropRatio(List<String> cropRatios, String ratio) {
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/contentsettings/ContentSettingsService.java]: Unit tests?

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**